other/res/restrict-node-label-changes/03-check.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+- script: |
```

Comment:
I would also like to see an addition, modification, and removal of another label aside from `foo`.

Reply:
Okayy, added.